itest/rust/src/object_tests/property_test.rs

```diff
@@ -453,6 +465,12 @@ fn export_resource() {
 #[derive(GodotClass)]
 #[class(init)]
 struct ExportOverride {
+    #[export_group(name = "some group", prefix = "a_")]
+    #[export]
+    a_first: i32,
+    #[export_subgroup(name = "some subgroup", prefix = "b_")]
+    #[export]
+    b_second: i32,
```

Comment:
Please use empty lines for grouping (no pun intended):
```suggestion
    #[export_group(name = "some group", prefix = "a_")]
    #[export]
    a_first: i32,

    #[export_subgroup(name = "some subgroup", prefix = "b_")]
    #[export]
    b_second: i32,

```